src/views/Article/post.jsx

```diff
@@ -96,11 +100,11 @@ class Post extends Component {
                         : null }
                     </div>
                     <div className="col">
-                      { CurrentUser === post[item].author.username ? <i className="fa fa-trash fa-1x" style={{color: 'Tomato'}} onClick={this.onDeleteClick.bind(this, post[item].slug)} aria-hidden="true"> Delete</i>
-                        : null }
-                      
+                      <Link to="#null">
+                        { CurrentUser === post[item].author.username ? <i className="fa fa-trash fa-1x" style={{color: 'Tomato'}} onClick={this.onDeleteClick.bind(this, post[item].slug)} aria-hidden="true"> Delete</i>
```

Comment:
Hello @Amoswachira ,
You can bind the event handler in the constructor instead of binding it during render. In addition, if you use ES6 methods with the fat arrow functions, it will be bound without having to explicitly do the binding.

Reply:
@Ogutu-Brian its is fixed